composer.patches.json

```diff
@@ -22,7 +22,8 @@
       "Provide a list of unconfigured but eligible fields - https://www.drupal.org/project/linkchecker/issues/3244743": "PATCHES/linkchecker-unconfirmed-but-eligible-field-list-3244743.patch",
       "Do not break admin denied": "./PATCHES/linkchecker_use_uid.patch",
       "Avoid null links": "./PATCHES/linkchecker_null_link.patch",
-      "Recognize previous revisions https://www.drupal.org/project/linkchecker/issues/3366753": "./PATCHES/linkchecker-previous-revisions-3366753.patch"
+      "Recognize previous revisions https://www.drupal.org/project/linkchecker/issues/3366753": "./PATCHES/linkchecker-previous-revisions-3366753.patch",
+      "Fallback to GET if HEAD fails - https://www.drupal.org/project/linkchecker/issues/3334240": "https://git.drupalcode.org/project/linkchecker/-/merge_requests/56.diff"
```

Comment:
Not a blocker, but I think it's better to copy the diff to a .patch in PATCHES/, until https://humanitarian.atlassian.net/browse/OPS-10181 is resolved.